### PR TITLE
Bump release link on README.adoc

### PR DESCRIPTION
**Pull request by Victoremepunto**

### CODE REVIEW

**Comment by bsquizz** [comment]
I think tests failed because you did not sign the commit, or your gpg key is not present in your app-sre users file.

**Review by bsquizz — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -115,7 +115,7 @@ shown below:
 [source,shell]
 ----
 # Be sure to get the latest release in the link above!
-minikube kubectl -- apply -f https://github.com/RedHatInsights/clowder/releases/download/0.14.0/clowder-manifest-0.14.0.yaml --validate=false
+minikube kubectl -- apply -f https://github.com/RedHatInsights/clowder/releases/download/0.15.0/manifest.yaml --validate=false
```
Comment (psav) [inline README.adoc:118]: ```clowder-manifest-0.15.0.yaml``` has now been created

**Review by psav — CHANGES_REQUESTED** [review]

**Comment by Victoremepunto** [comment]
@bsquizz  thank you man, I pushed my commit, now signed and that looks like worked :)

File: README.adoc (hunk 1 above)
Comment (Victoremepunto) [reply README.adoc]: fixed

**Pull request merged**